MC/bin/o2dpg_sim_workflow.py

```diff
@@ -1163,7 +1165,7 @@ def getDigiTaskName(det):
 
    PVFINDERtask = createTask(name='pvfinder_'+str(tf), needs=pvfinderneeds, tf=tf, cwd=timeframeworkdir, lab=["RECO"], cpu=NWORKERS, mem='4000')
    PVFINDERtask['cmd'] = '${O2_ROOT}/bin/o2-primary-vertexing-workflow ' \
-                         + getDPL_global_options() + putConfigValuesNew(['ITSAlpideParam','MFTAlpideParam', 'pvertexer', 'TPCGasParam'], {"NameConf.mDirMatLUT" : ".."})
+                         + getDPL_global_options() + putConfigValuesNew(['ITSAlpideParam','MFTAlpideParam', 'pvertexer', 'TPCGasParam', 'TPCCorrMap'], {"NameConf.mDirMatLUT" : ".."})
```

Comment:
Note that also SVertexer reads this object from the CCDB though at the moment does not use it. Could you change https://github.com/AliceO2Group/O2DPG/blob/181535bdaa01177b3585d222e0d4341f7625b523/MC/bin/o2dpg_sim_workflow.py#L1324C4-L1324C29 to 
```
   SVFINDERtask['cmd'] += getDPL_global_options(bigshm=True) + svfinder_threads + putConfigValuesNew(['svertexer', 'TPCCorrMap'], {"NameConf.mDirMatLUT" : ".."})
```
?